exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Schema.java

```diff
@@ -27,8 +28,11 @@
  * the core automatically tracks every Merkelized collection used by the user
  * services. It aggregates state hashes of these collections into a single
  * Merkelized meta-map. The hash of this meta-map is considered the hash of the
- * entire blockchain state and is recorded as such in blocks and Precommit
- * messages.
+ * entire blockchain state and is recorded as such in {@linkplain Block#getStateHash() blocks}
+ * and Precommit messages.
+ *
+ * <p>Exonum starts aggregating a service collection state hash once it is <em>initialized</em>:
```

Comment:
```suggestion
 * <p>Exonum starts aggregating a service collections state hash once it is <em>initialized</em>:
```

Reply:
The sentence says when Exonum starts aggregating one (any, a single) collection.